setup.cfg

```diff
@@ -79,6 +79,8 @@ ignore =
     sklearn/metrics/_pairwise_distances_reduction/_datasets_pair.pyx
     sklearn/metrics/_pairwise_distances_reduction/_gemm_term_computer.pxd
     sklearn/metrics/_pairwise_distances_reduction/_gemm_term_computer.pyx
+    sklearn/metrics/_pairwise_distances_reduction/_pairwise_distances.pxd
+    sklearn/metrics/_pairwise_distances_reduction/_pairwise_distances.pyx
```

Comment:
Side-note: it would be nice not to have duplication with identical lines in `.gitignore`, yet I am not sure.